src/Fixer/PhpTag/BlankLineAfterOpeningTagFixer.php

```diff
@@ -57,6 +64,19 @@ public function isCandidate(Tokens $tokens): bool
         return $tokens->isTokenKindFound(T_OPEN_TAG);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
+    {
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('blank_line', 'Ensure after linebreak there is. there isn\'t or ignore the empty line.'))
+                ->setAllowedValues(['single', 'none', null])
```

Comment:
i don't like the `null` option. one doesn't like the rule, let's disable the rule rather than configuring option with `null`